src/types/index.ts

```diff
@@ -566,6 +565,14 @@ export interface IPopupConfig {
   txBase64?: Encoded.Transaction;
 }
 
+export interface TokenProps {
+  contractId?: string;
+  tokenPairs?: any; // TODO: find and declare types
```

Comment:
```suggestion
  tokenPairs?: Record<'token0' | 'token1', IToken | null>;
```

I suggest to create a `TokenPair` type out of this and use it in `src/popup/pages/FungibleTokens/TokenContainer.vue` file (line 202).

Reply:
Using this type here creates many ts issues in `TokenDetails.vue`

Reply:
OK